packages/theme/index.ts

```diff
@@ -38,6 +38,7 @@ export type StarryUIThemeFacet =
  | 'row'
  | 'tray'
  | 'tray-spacer'
+ | 'grid'
```

Comment:
You added the `grid` theme facet, but you aren't using it in the `grid` component - you'll want to use the theme to apply styles using the facets, avoid using `.style.* = `.

Reply:
You'll also need to add styles to `@starryui/theme-midnight` and `@starryui/theme-sandstone` to support the styles needed by the grid.

Reply:
@Vikulakav we should use the facet, just add one for the grid container, and one for the grid cell. Then you can connect those elements to the facet with:

```
attachThemeFacet(
 element: HTMLElement,
 theme: StarryUITheme,
 facet: StarryUIThemeFacet
) 
```

For an example, this one is adding the `'button'` facet to a div to make a div look like a button in the currently loaded `theme` is in `main-tray.ts` line 67:

```
attachThemeFacet(themeSwitcher, themeMidnight, 'button')
```

This is what gives the theme switcher in the docs site in the upper right hand corner its CSS styles. Let me know if you have more questions on this.